contracts/UtilityBrandedToken.sol

```diff
@@ -58,7 +57,7 @@ contract UtilityBrandedToken is UtilityToken {
      * @param _organization Address of the Organization contract.
      */
     constructor(
-        EIP20Interface _token,
+        address _token,
```

Comment:
Yes, UtilityToken contract accepts the `_token` in `address` type. So I think changing the type of `_token`  to `address` at the very initial stage is better. It will prevent unnecessary typecasting of it to `address`.